### DISCUSSION TITLE
Redesign the user registration team flows

### DISCUSSION THREAD

**Discussion by jesperordrup** [coollabsio/coolify#501] [Feature Requests]
Allow teams admin to add new users that are not already registered. 

The current flow is probably based on that a coolify server is open for public to register teams and start using. Paas style. I bet that many just want to use it as setup for themselves and a team thus disabling registration.

To support "own server" and teams it makes sense to allow admins to work with team users without having to open for registration.

Related, it seems relevant to control access more granually:
https://github.com/coollabsio/coolify/discussions/500

![image](https://user-images.githubusercontent.com/3120151/179931252-68fc69f3-ef2d-44f6-9b1b-af61f94f8a11.png)

ps. In any case the team page itself needs a bit of design (move the "send invitation" below the email field).

**Comment by Fronix** [comment]
What's also missing is a way to easier manage teams. As it is right now if I set up an application and I want some other members to have access to those applications, I cannot move the application to that team. I instead have to open registration, let them register, then invite them to my admin team. Or I have to delete the application and create them in another team using another account.

**Comment by simonjcarr** [comment]
Is there any update on this feature.

Allow admin to create a user and either set an initial password or send an invitation that allows a user to setup there account and password, without having to open the server for anyone to register an account.
